crates/aptos/src/common/types.rs

```diff
@@ -53,13 +50,18 @@ use std::{
     time::{Duration, Instant, SystemTime, UNIX_EPOCH},
 };
 use thiserror::Error;
+use crate::common::types::KeyType::X25519;
 
 pub const USER_AGENT: &str = concat!("aptos-cli/", env!("CARGO_PKG_VERSION"));
 const US_IN_SECS: u64 = 1_000_000;
 const ACCEPTED_CLOCK_SKEW_US: u64 = 5 * US_IN_SECS;
 pub const DEFAULT_EXPIRATION_SECS: u64 = 30;
 pub const DEFAULT_PROFILE: &str = "default";
 
+// Custom header to identify the client
+const X_APTOS_CLIENT_HEADER: &str = "x-aptos-client";
```

Comment:
Let's use the constant we've already got in the API types: https://github.com/aptos-labs/aptos-core/blob/main/api/types/src/headers.rs#L21.